sql/util/requests.sql

```diff
@@ -178,21 +167,34 @@ _cdn_provider: $._cdn_provider,
 _gzip_save: $._gzip_save,
 type: prettyType,
 ext: ext,
-format: getFormat(prettyType, mimeType, ext)
-    };
+format: getFormat(prettyType, mimeType, ext,),
+protocol: $._protocol,
+pushed: $._was_pushed,
+tls_version: $._tls_version,
+tls_cipher_suite: $._tls_cipher_suite,
+cert_issuer: securityDetails.issuer,
+cert_keyexchange: securityDetails.keyExchange,
+cert_cipher: securityDetails.cipher,
+cert_protocol: securityDetails.protocol
 
+    };
     return summary;
   } catch (e) {
     return e
   }
 """;
 
 SELECT
-  CAST('2020-08-01' AS DATE) AS date,
+  CAST('2021-07-01' AS DATE) AS date,
   _TABLE_SUFFIX AS client,
   page,
-  url,
-  getSummary(payload).*, --  noqa: PRS, L013
+  rank,
+  req.url as url,
```

Comment:
```suggestion
  url,
```

Reply:
`url` would be ambiguous here, which is why I needed to use `req.url`

Reply:
To disambiguate I suggested aliasing `pages.url` to `page`

Reply:
I changed it to a `USING` join instead to resolve this.